source/view.js

```diff
@@ -344,21 +344,19 @@ view.View = class {
             this._events = {};
             this._events.scroll = (e) => this._scrollHandler(e);
             this._events.wheel = (e) => this._wheelHandler(e);
-            this._events.mousedown = (e) => this._mouseDownHandler(e);
             this._events.gesturestart = (e) => this._gestureStartHandler(e);
-            this._events.touchstart = (e) => this._touchStartHandler(e);
+            this._events.pointerdown = (e) => this._pointerDownHandler(e);
         }
         const graph = this._element('graph');
         if (graph) {
             graph.focus();
         }
         graph.addEventListener('scroll', this._events.scroll);
         graph.addEventListener('wheel', this._events.wheel, { passive: false });
-        graph.addEventListener('mousedown', this._events.mousedown);
         if (this._host.environment('agent') === 'safari') {
```

Comment:
Due to this condition, Safari doesn't use pointerdown event.

The pointerdown event is supported from Safari 13.
https://developer.mozilla.org/en-US/docs/Web/API/Element/pointerdown_event#browser_compatibility

Safari 13 was released on September 20, 2019
https://en.wikipedia.org/wiki/Safari_(web_browser)